xmake/toolchains/llvm/xmake.lua

```diff
@@ -32,7 +32,7 @@ toolchain("llvm")
     set_toolset("cc",     "clang")
     set_toolset("cxx",    "clang", "clang++")
     set_toolset("mxx",    "clang", "clang++")
-    set_toolset("mm",    "clang", "clang++")
+    set_toolset("mm",     "clang")
```

Comment:
and here. clang++ cannot compile objc? why?

Reply:
you didn't add clang++ for cc so i did the same for objc